docs/metrics/customizing-the-sdk/README.md

```diff
@@ -403,6 +403,61 @@ AnotherFruitCounter.Add(4, new("name", "mango"), new("color", "yellow")); // Not
 streams. There is no ability to apply different limits for each instrument at
 this moment.
 
+### Exemplars
+
+Exemplars are example data points for aggregated data. They provide access to
+the raw measurement value, time stamp when measurement was made, and trace
+context, if any. It also provides "Filtered Tags", which are those attributes
+(Tags) [dropped by a view](#select-specific-tags). Exemplars are an opt-in
```

Comment:
Rephrase to "It also provides "Filtered Tags", which are attributes (Tags) that are [dropped by a view](#select-specific-tags)."